Common/TDSQASystemAPI/DAL/configs/daos/AccommodationFamilyDAO.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections.Generic;
+using TDSQASystemAPI.DAL.configs.dtos;
+
+namespace TDSQASystemAPI.DAL.configs.daos
+{
+    /// <summary>
+    /// A class for saving <code>AccommodationFamilyDTO</code>s to the <code>OSS_Configs..Client_AccommodationFamily</code> 
+    /// table
+    /// </summary>
+    public class AccommodationFamilyDAO : TestPackageDaoBase<AccommodationFamilyDTO>
+    {
+        public AccommodationFamilyDAO()
+        {
+            DbConnectionStringName = DatabaseConnectionStringNames.CONFIGS;
+            TvpType = "AccommodationFamilyTable";
+            InsertSql =
+                "INSERT \n" +
+                "   dbo.Client_AccommodationFamily (ClientName, Family, Label) \n" +
+                "SELECT \n" +
+                "   ClientName, \n" +
+                "   Family, \n" +
+                "   Label \n";
+        }
+
+        public override void Insert(IList<AccommodationFamilyDTO> recordsToSave)

```

Comment:
For the subtypes of `TestPackageDaoBase`, why does the `Insert` methods need to be overridden?

Reply:
Unlike Java, methods in C# are not `virtual` by default.  If you don't add the `override` keyword, the IDE complains that you're hiding the method of the same name in the base class.  Had I used a different method name, I wouldn't have had to use `override`.  Also, I could've used `public new Insert(...) {}` to overwrite the base class's `Insert` implementation, but that seemed like overkill.